bedrock/firefox/urls.py

```diff
@@ -89,6 +89,10 @@
     url(tracking_protection_re, views.TrackingProtectionTourView.as_view(),
         name='firefox.tracking-protection-tour.start'),
 
+    url(r'^firefox/features/adblocker/$', views.ad_blocker, name='firefox.features.adblocker'),
```

Comment:
The url shouldn't need defining twice here, the `page` definition below would be enough. Using a `page` definition also means you don't need to define a view 😄